Lib/pathlib/__init__.py

```diff
@@ -5,8 +5,8 @@
 operating systems.
 """
 
-from ._os import *
+from ._abc import *
 from ._local import *
```

Comment:
Minor (and unrelated), but out of curiosity why do we use relative imports here?

Reply:
I copied it from `asyncio/__init__.py` heh. What would you prefer?

Reply:
A fun survey:

* Both absolite and relative same-package imports: `concurrent.futures`, `importlib`, `json`, `multiprocessing`, `tkinter`, `wsgiref`, `xml`.
* Absolute same-package imports: `ctypes`, `email`, `html`, `htttp`, `idlelib`, `sqlite3`, `sysconfig`, `turtledemo`, `urllib`, `xmlrpc`.
* Relative same-package imports: `_pyrepl`, `asyncio`, `curses`, `encodings`, `re`, `tomllib`, `unittest`, `venv`, `zipfile`, `zoneinfo`.

I was going to suggest absolute imports, per [PEP 8](https://peps.python.org/pep-0008/#imports) "Absolute imports are recommended, as they are usually more readable and tend to be better behaved". However it seems that current practice is all over the place!

Reply:
:) thank you for the advice! I've adjusted the patch to use absolute imports now.